src/index.html

```diff
@@ -94,7 +94,7 @@ <h3>Content & Design</h3>
   <div class="row">
     <div class="col-8">
       <h2>Sold! How do I get in on this?</h2>
-      <p>Registration will open on <strong>Sep 5, 2022</strong> at this location. Craving for more information? Sign up for updates here:</p>
+      <p>Registration will open on <strong>Sep 5, 2022</strong> at this location. Craving for more information? Sign up for updates here: <a href="https://lists.ubuntu.com/mailman/listinfo/summit-news">https://lists.ubuntu.com/mailman/listinfo/summit-news</a></p>
```

Comment:
I don't think we should show raw URLs like that, it doesn't look nice and will mean very little to screen readers - plus, we basically have a CTA there in the text:

```suggestion
      <p>Registration will open on <strong>Sep 5, 2022</strong> at this location. Craving for more information?  <a href="https://lists.ubuntu.com/mailman/listinfo/summit-news">Sign up for updates here</a>.</p>
```